types/fuzzy-search/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for fuzzy-search 2.1
+// Project: https://github.com/wouter2203/fuzzy-search#readme
+// Definitions by: Alex Deas <https://github.com/alex-deas>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+declare class FuzzySearch<T> {
+    constructor(haystack: T[], keys?: string[], options?: FuzzySearch.Options);
+    search(needle: string): T[];
```

Comment:
needle should be optional here as per https://github.com/wouter2203/fuzzy-search#documentation